metadataserver/types/annotations.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+const (
+	IamRoleArnAnnotation                = "iam.amazonaws.com/role"
+	RequireTokenAnnotation              = "imds.titus.netflix.com/requireToken" // nolint:gosec
```

Comment:
why does this trigger gosec?

Reply:
metadataserver/types/annotations.go:5:2: G101: Potential hardcoded credentials (gosec)
	RequireTokenAnnotation              = "imds.titus.netflix.com/requireToken"